spec/features/push_flow_spec.rb

```diff
@@ -48,7 +48,9 @@
                                          Cangaroo::WarehouseJob,
                                          Cangaroo::ShippedOrderMailJob]
 
-    post endpoint_index_path, params: store_payload, headers: headers
+    # NOTE we need to use positional parameters here to keep these tests
+    # compatible with Rails 4.2.x
+    post endpoint_index_path, store_payload, headers
```

Comment:
Thanks for pointing this problem! 
As i understand, this behavior will fail on Rails 5.1:

```
DEPRECATION WARNING: Using positional arguments in integration tests has been deprecated,
in favor of keyword arguments, and will be removed in Rails 5.1.
```

We should drop support for Rails 4.2 or add a conditional statement for Rails 5+ (I prefer to add a conditional statement).

```
if Rails::VERSION::MAJOR >= 5
  post endpoint_index_path, params: store_payload, headers: headers
else
  post endpoint_index_path, store_payload, headers
end
```

As see also that we have no Appraisal for rails 5.1 :
https://github.com/nebulab/cangaroo/blob/master/Appraisals

Reply:
Ah, good point, @bricesanchez! Let me see what I can do.